modules/configuring-firewall.adoc

```diff
@@ -5,16 +5,99 @@
 [id="configuring-firewall_{context}"]
 = Configuring your firewall for {product-title}
 
-Before you install {product-title}, you must configure your firewall to access
-Red Hat Insights.
+Before you install {product-title}, you must configure your firewall to grant access to the sites that {product-title} requires.
 
 .Procedure
 
-* Set your firewall to allow the following host names and ports on the outgoing
-network firewall:
+. Whitelist the following registry URLS:
 +
-----
-cert-api.access.redhat.com:443
-api.access.redhat.com:443
-infogw.api.openshift.com:443
-----
+[cols="3,4",options="header"]
+|===
+|URL | Function
+|`registry.redhat.io`
+|Provides core container images
+
+|`quay.io`
```

Comment:
Change this to *.quay.io to allow region failover and the future cdn.quay.io (see below comments about first party image serving).